src/data/verses/verse3829.ts

```diff
@@ -27,6 +28,7 @@ const verseWords: Word[] = [
     pageNumber: 455,
     lineNumber: 4,
     text: 'كِتَٰبٌ',
+    verse: undefined,
```

Comment:
Please see my comment regarding attaching the entire `verse` object to each word.